v03_pipeline/lib/tasks/write_metadata_for_run.py

```diff
@@ -75,15 +75,20 @@ def run(self) -> None:
             'callsets': self.callset_paths,
             'run_id': self.run_id,
             'sample_type': self.sample_type.value,
-            'families': {},
-            'families_failed_missing_samples': {},
-            'families_failed_relatedness_check': {},
-            'families_failed_sex_check': {},
+            'family_samples': {},
```

Comment:
I renamed these fields to be consistent with this structure across the tables.  It'll require a change to the check-new-samples job though.

Reply:
can you point me to an example of a run with the updated metadata?

Reply:
I'd need to actually run this first, but I can when we have it!